src/drive/lib/doctypes.js

```diff
@@ -2,6 +2,7 @@ import { Contact, Group } from 'models'
 import extraDoctypes from 'drive/lib/extraDoctypes'
 
 export const DOCTYPE_FILES = 'io.cozy.files'
+export const DOCTYPE_FILES_SETTINGS = 'io.cozy.files.settings'
```

Comment:
can you add the info in cozy-doctypes please ?

Reply:
Oh right, forgot it

Reply:
https://github.com/cozy/cozy-doctypes/pull/155